extensions/2.0/Vendor/EXT_mesh_primitive_restart/schema/primitiveGroup.schema.json

```diff
@@ -0,0 +1,40 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
```

Comment:
Nitpick: Upgrade to JSON schema draft 2020-12 to match glTF schemas.

Reply:
Done.